### DISCUSSION TITLE
Finished the assignment but not able to figure out pull request

### DISCUSSION THREAD

**Discussion by Pratik-A-N** [open-sauced-craftwork/browser-extension#46] [Q&A] [answered]
![Screenshot from 2023-03-26 02-01-32](https://user-images.githubusercontent.com/91190547/227740204-f892cb7f-ae27-4762-b7ea-fa132576e9ae.png)

I have pushed the code in the development branch and I'm able to see the open pull request button but when I click on that it just shows me the comparison of changes I have made.

@bdougie can you please help me with what should I do next or only this much work only needed??

**Comment by nickytonline** [comment, marked as answer]
You’ll need to click the Compare & Pull Request button to create the pull request. See https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request#
